### ISSUE TITLE
attention_mask in RITA is diffenrent from it in other Huggingface transformer models

### ISSUE THREAD

**Issue by zzhongzz** [lightonai/RITA#2]
The `attention_mask` in Huggingface transformer models is padding masking, where 1 means not masked and 0 means masked. (https://huggingface.co/docs/transformers/main/en/glossary#attention-mask)
I think in RITA model it is named as `padding_mask`. 

However, `RITAModelForCausalLM` class does not have the argument `padding_mask`. Instead, it passes `attention_mask` argument into `DecoderLayer` class like [this](https://huggingface.co/lightonai/RITA_s/blob/main/rita_modeling.py#L251): `x = layer(x, attn_mask=attention_mask)`, where `attn_mask` in RITA is the causual mask instead of padding mask. This will lead to unexpected result.

So, I think it might be better to keep the same API as other Huggingface transformer models. i.e. let users pass the Huggingface-style `attention_mask` into the model as padding mask.

Thank you!

**Comment by DanielHesslow** [comment]
Thanks, yeah it's definitely best to keep the same naming schemes as in standard HF models. I'll have a look, will update as soon as I get the bandwidth.

**Comment by DanielHesslow** [comment]
Okay, it's updated now. 
- renamed attention_mask -> causal_mask
- renamed padding_mask -> attention_mask
- inverted the meaning of the padding_mask/now attention_mask, so 1 means not not masked.
